app/js/directives/yesNoQuestion.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('ddsApp').directive('yesNoQuestion', function() {
+    return {
+        restrict: 'E',
+        transclude: true,
+        templateUrl: 'partials/foyer/yes-no-question.html',
+        scope: true,
+        controller: 'yesNoQuestionCtrl',
+        link: function ($scope, $element, $attributes) {
+            $scope.modelString = $attributes.model;
```

Comment:
Should it really be called `ng-model` then, or is it misleading?

(I don't know exactly what `ng-model` entails, that's a real question)